client/src/components/Notification/InWebNotification.tsx

```diff
@@ -0,0 +1,17 @@
+import { useSocket } from '../../context/useSocketContext';
+import { useState } from 'react';
+import { useSnackBar } from '../../context/useSnackbarContext';
+
+export default function InWebNotification(): JSX.Element {
```

Comment:
Can I get a clarification on what does "In Web Notification" mean?

Reply:
Oh my bad, I was supposed to remove that component. It was only for the test